apps/ai-image-generator/backend/src/utils.ts

```diff
@@ -19,17 +24,31 @@ export const toSharp = (imageStream: NodeJS.ReadableStream): sharp.Sharp => {
   return imageStream.pipe(sharpStream);
 };
 
+const difference = (a: number, b: number): number => {
+  return Math.abs(a - b);
+};
+
 // note: we don't care about alpha channel here, only the RGB color
```

Comment:
nit: I suppose we can remove this note now that we added the conditional to compare alpha